Makefile

```diff
@@ -806,4 +809,12 @@ act_list: check_act ## List all github actions that can be executed locally with
 act_reviewdog: check_act check_gh ## Run the reviewdog workflow locally like so: `GITHUB_TOKEN=$(gh auth token) make act_reviewdog`
 	$(eval CONTAINER_ARCH := $(shell make -s detect_arch))
 	@echo "Detected architecture: $(CONTAINER_ARCH)"
-	act -v -s GITHUB_TOKEN=$(GITHUB_TOKEN) -W .github/workflows/reviewdog.yml --container-architecture $(CONTAINER_ARCH)
\ No newline at end of file
+	act -v -s GITHUB_TOKEN=$(GITHUB_TOKEN) -W .github/workflows/reviewdog.yml --container-architecture $(CONTAINER_ARCH)
+
+#############################
+### Grove Gateway Helpers ###
+#############################
+
+.PHONY: grove_eth_balance
+grove_eth_balance: # Must have GROVE_PORTAL_APP_ID environment variable set
+	curl $(GROVE_GATEWAY)/v1/$(GROVE_PORTAL_APP_ID) -H 'Content-Type: application/json' --data $(GROVE_TEST_RELAY)
```

Comment:
This simply uses grove's mainnet on Morse.

Our goal is to force a request using a Shannon relay. How can we do that?

Reply:
The content of the "protocol" field in the JSON-RPC data determines which network the gateway uses for the relay. I have added a comment in the Makefile so it is clear.